web/client/actions/security.js

```diff
@@ -18,13 +22,14 @@ const CHANGE_PASSWORD = 'CHANGE_PASSWORD';
 const CHANGE_PASSWORD_SUCCESS = 'CHANGE_PASSWORD_SUCCESS';
 const CHANGE_PASSWORD_FAIL = 'CHANGE_PASSWORD_FAIL';
 const LOGOUT = 'LOGOUT';
-
+const REFRESH_ACCESS_TOKEN = 'REFRESH_ACCESS_TOKEN';
 
 function loginSuccess(userDetails, username, password, authProvider) {
     return {
         type: LOGIN_SUCCESS,
         userDetails: userDetails,
         // set here for compatibility reasons
+        // TODO: verify if the compatibility reasons still hold and remove otherwise
         authHeader: 'Basic ' + btoa(username + ':' + password),
         username: username,
         password: password,
```

Comment:
there are other places (in reducers) where the authHeader is computed. Don't know why, hence the question on "compatibility reasons"